ocaml/xapi-consts/api_messages.ml

```diff
@@ -353,3 +353,31 @@ let failed_login_attempts = addMessage "FAILED_LOGIN_ATTEMPTS" 3L
 
 let tls_verification_emergency_disabled =
   addMessage "TLS_VERIFICATION_EMERGENCY_DISABLED" 3L
+
+exception InvalidMessagePrio of int64
+
+let css_expiring prio =
```

Comment:
`css` is a specific term of Citrix. I understand it correspond the `Updates` feature.
Could it be re-named to something like `updates_feature_expiring`?